core/editor/loader/v2/editor-v2-loader.php

```diff
@@ -198,24 +171,17 @@ public function print_root_template() {
 	}
 
 	private function get_packages_to_enqueue() : array {
-		$experiments_manager = Plugin::$instance->experiments;
+		$packages_to_enqueue = Editor_V2_Packages::collect();
```

Comment:
Maybe add self::LIBS here and remove the array_merge from line 36

Reply:
I think they should be separated.